src/core/Microsoft.PowerToys.Settings.UI/ViewModels/PowerLauncherViewModel.cs

```diff
@@ -1,13 +1,178 @@
-﻿using System;
-
-using Microsoft.PowerToys.Settings.UI.Helpers;
-
-namespace Microsoft.PowerToys.Settings.UI.ViewModels
-{
-    public class PowerLauncherViewModel : Observable
-    {
-        public PowerLauncherViewModel()
-        {
-        }
-    }
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Text.Json;
+
+using Microsoft.PowerToys.Settings.UI.Helpers;
+using Microsoft.PowerToys.Settings.UI.Lib;
+using Microsoft.PowerToys.Settings.UI.Views;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.PowerToys.Settings.UI.ViewModels
+{
+    public class PowerLauncherViewModel : Observable
+    {
+        public PowerLauncherSettings settings;
+        private const string POWERTOY_NAME = "PowerLauncher";
+
+
+        public PowerLauncherViewModel()
+        {
+            if (SettingsUtils.SettingsExists(POWERTOY_NAME))
+            {
+                settings = SettingsUtils.GetSettings<PowerLauncherSettings>(POWERTOY_NAME);
+            } else
+            {
+                settings = new PowerLauncherSettings();
+            }
+        }
+
+        private void UpdateSettings([CallerMemberName] string propertyName = null)

```

Comment:
nit: public methods should come before private methods (its a stylecop thing, I actually do not have any preference on this) we are gonna have to apply these changes once stylecop kicks in. Since we are already doing that in another project, I would advise adding stylecop just to format the code you added and then remove again. This will avoid us having to format this code again when we add in stylecop.

Reply:
I think its easier to delay this merge until the stylecop PR is merged to dev/settingsV2, then rebase these changes on top of it.